Documentation/ApiOverview/Categories/Index.rst

```diff
@@ -132,3 +132,16 @@ for rendering. Check out the
 
 The HMENU object also has a :ref:`"categories" special type <t3tsref:hmenu-special-categories>`,
 to display a menu based on categorized pages.
+
+User permissions for system categories
+======================================
+
+In most aspects system categories are treated like any other record. They can
+be viewed or edited by editors if they are stored in a folder where the editor
+has access to and if the table :sql:`sys_category` if allowed in the field
```

Comment:
.. and if the table :sql:`sys_category` is allowed in the field ... of an assigned user group

Reply:
You can make suggestions ready for commit. ;-)

Reply:
Uh.. never realized that. Thanks for the info. (I'm rarely doing reviews on github, it still does not feel "right" 😀 )